api/src/dtos/applications/demographic.dto.ts

```diff
@@ -5,13 +5,6 @@ import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum
 import { AbstractDTO } from '../shared/abstract.dto';
 
 export class Demographic extends AbstractDTO {
-  @Expose()
```

Comment:
There are other places in the app where we are maintaining references to ethnicity, so this is causing type errors. We should either (1) remove references everywhere, which would including changes across public, partners, shared-helpers, and the backend, or (2) keep it in the data model and just remove it from the csv export, which is what I would recommend doing here for now.

Reply:
Do you think we should have a follow up ticket to do proper removal of the field across doorway?

Reply:
I think it's worth making the ticket!